blank-canvas-blocks/sass/blocks/_image.scss

```diff
@@ -0,0 +1,15 @@
+.wp-block-image {
+	/* 
+	From what I can tell the below are styles regularly used by themes
+	to fix the image block.  I believe these should go into the block's
+	default styles.  It's difficult to say how this will land, however 
+	based on discussion found in (many) related issues here: 
+	https://github.com/WordPress/gutenberg/issues/28923
+
+	Further gutenberg APPEARS to be defaulting the margin to the same as 
+	the below, however (for reason's that aren't clear to me now) this
+	isn't coming through.
+	*/
+	text-align: center;
```

Comment:
I think this should be customizable in the theme.json - `--wp-custom-image-alignment`

Reply:
@jffng and I reviewed and because of image caption alignment decided not to offer configurable alignment (yet).  This could possibly be covered in the future if we want to configure that in a theme.